ui/shared/components/panel/sample.jsx

```diff
@@ -33,9 +33,16 @@ const Sample = React.memo(({ loadedSample, isOutdated, hoverDetails }) => (
         }
       </span>
     }
-    content={loadedSample ?
-      `data was${isOutdated ? ' previously ' : ''} ${hoverDetails ? `${hoverDetails} on ${new Date(loadedSample.loadedDate).toLocaleDateString()}` : 'loaded'}` :
-      'no data available'}
+    content={
+      <div>
+        {loadedSample ?
+          `data was${isOutdated ? ' previously ' : ''} ${hoverDetails ? `${hoverDetails} on ${new Date(loadedSample.loadedDate).toLocaleDateString()}` : 'loaded'}` :
+          'no data available'}
+        <VerticalSpacer height={5} />
+        {loadedSample.sampleType && loadedSample.sampleType === SAMPLE_TYPE_RNA &&
```

Comment:
`rnaSeqTypes` will only have values for RNA samples so you can remove this part of the condition check

Reply:
Without this part of the check some old(?) rna samples which do not have any matches in the 3 rnaseqtype tables will display like this:
<img width="390" alt="image" src="https://github.com/broadinstitute/seqr/assets/25799914/aeabeb46-3701-4504-b2c1-f73c56148f8f">

There are 76 of these samples
```
WITH sample_rnaseq as (
    select s.guid, s.id,
           seqr_rnaseqtpm.id as tpm_id, seqr_rnaseqoutlier.id as outlier_id, sr.id as splice_outlier_id
    from seqr_sample s
    left join seqr_rnaseqtpm on s.id = seqr_rnaseqtpm.sample_id
    left join seqr_rnaseqoutlier on s.id = seqr_rnaseqoutlier.sample_id
    left join public.seqr_rnaseqspliceoutlier sr on s.id = sr.sample_id
    where s.sample_type in ('RNA')
)
SELECT * FROM sample_rnaseq
WHERE tpm_id is null
and outlier_id is null
and splice_outlier_id is null


```

Reply:
they're all probably from the R0246_rna_seq_bc project, I've been using that for testing since I was certain it would have rna samples given the name

Reply:
Oh we should only be showing this for active samples, not inactive sample